frontend/src/components/common/Icon/types.ts

```diff
@@ -4,7 +4,7 @@ export type IconProps = {
   id?: string;
   className?: string;
   ariaLabel?: string;
-  icon: 'Authenticate' | 'Exit';
+  icon: 'Authenticate' | 'Exit' | 'JSON';
```

Comment:
consider making this a new type that can be exported for other components to use